build.sbt

```diff
@@ -109,10 +109,10 @@ lazy val java = project.in(file("faunadb-java"))
     testOptions += Tests.Argument(TestFrameworks.JUnit, "+q", "-v"),
     apiURL := Some(url("http://faunadb.github.io/faunadb-jvm/faunadb-java/api/")),
 
-    (javacOptions in doc) := Seq("-source", "1.7",
+    (javacOptions in doc) := Seq("-source", "1.7", "-Xdoclint:none",
```

Comment:
I understand that the doclint is pretty aggressive, but were our docs not linting properly?